sklearn/ensemble/tests/test_bagging.py

```diff
@@ -288,7 +288,8 @@ def test_probability():
                                   np.exp(ensemble.predict_log_proba(X_test)))
 
         # Degenerate case, where some classes are missing
-        ensemble = BaggingClassifier(base_estimator=LogisticRegression(),
+        ensemble = BaggingClassifier(base_estimator=LogisticRegression(
+                                          solver='lbfgs'),
```

Comment:
Sorry for not being clear.

By breaking code, I mean people already using `ensemble = BaggingClassifier(base_estimator=LogisticRegression()` in their software, wouldn't be too pleased to see an error right now.